packages/front-end/services/experiments.ts

```diff
@@ -245,6 +245,25 @@ export function applyMetricOverrides(
   return { newMetric, overrideFields };
 }
 
+export function isExpectedDirection(
+  stats: SnapshotMetric,
+  metric: MetricInterface
+): boolean {
+  const expected: number = stats?.expected ?? 0;
+  return metric.inverse ? expected < 0 : expected > 0;
```

Comment:
This is a readability nit as I'm not a huge fan of logic nested in ternaries, but I think something like this would be more readable:

```ts
if (metric.inverse) {
  return expected < 0;
}

return expected > 0;
```

Reply:
I agree this is clearer. I'll make the change later today.